nebullvm/api/functions.py

```diff
@@ -37,7 +37,7 @@
 def _get_dl_framework(model: Any):
     if isinstance(model, torch.nn.Module):
         return DeepLearningFramework.PYTORCH
-    elif isinstance(model, tf.Module):
+    elif isinstance(model, tf.Module) and tf.Module != object:
```

Comment:
if you set `tf.Module` to `None` when tensorflow is not installed, you can avoid checking tf.Module type and just check if `model is not None`. Otherwise, we can simply check `model!=object` instead of `tf.Module != object`.